source/index.html.md.erb

```diff
@@ -54,16 +54,14 @@ refer users to.
 ## DNS
 
 * [BT DNS Change Process Pre-requisites](documentation/dns/BT-pre-requisites.html)
-* [How to check for domain activity before decommissioning](documentation/dns/check-domain-activity.html)
+* [Changes to Nameserver Records involving GDS Domains Team](documentation/dns/ns-changes-involving-gds.html)* [Domain Transfers for Non-gov.uk subdomains](documentation/dns/domain-transfer.html)* [Supporting migrations/DNS cutovers](documentation/dns/dns-cutovers.html)
```

Comment:
Are these meant to be on separate lines?

Reply:
Good call. Have separated out